padertorch/contrib/examples/audio_synthesis/wavenet/evaluate.py

```diff
@@ -0,0 +1,119 @@
+"""
+Example call:
+
+python -m padertorch.contrib.examples.audio_synthesis.wavenet.evaluate with exp_dir=/path/to/exp_dir
+"""
+import os
+from pathlib import Path
+import numpy as np
+import torch
+from padertorch import Model
+from padercontrib.database.librispeech import LibriSpeech
+from padertorch.contrib.examples.audio_synthesis.wavenet.data import prepare_dataset
+from sacred import Experiment, commands
+from scipy.io import wavfile
+
+from dlp_mpi import COMM, IS_MASTER, MASTER, split_managed
+
+from paderbox.io.new_subdir import get_new_subdir
+from paderbox.io import load_json, dump_json
+
+
+ex = Experiment('wavenet-eval')
+
+
+@ex.config
+def config():
+    exp_dir = ''
```

Comment:
Here you should use the get_new_storage_dir function from padertorch

Reply:
and probably call it storage_dir and add a storage observer from sacred

Reply:
exp_dir is required to be inputted by the user which is the path to the experiment directory with the model in it that is to be evaluated. get_new_subdir is used in line 40.